cli/cppcheckexecutor.cpp

```diff
@@ -418,7 +419,7 @@ bool CppCheckExecutor::reportSuppressions(const Settings &settings, const Suppre
 /*
  * That is a method which gets called from check_wrapper
  * */
-int CppCheckExecutor::check_internal(const Settings& settings) const
+int CppCheckExecutor::check_internal(const Settings& settings, Suppressions& supprs) const
```

Comment:
I would prefer the name suppressions is there a reason to use supprs instead?

Reply:
I did it to have less noise in the changes since it used to be `Settings::supprs`.

Reply:
crazy!

I understand. You can keep it this way then so it doesn't explode too much .. but feel free to change supprs to suppressions later in a separate PR.

Reply:
Might be a while though since changing that might cause (even more) merge conflicts in the PRs I have lined up.